runnumber-rest/rest.py

```diff
@@ -1,77 +1,20 @@
-import flask
-from flask import Flask
-
-from flask_restful import Api, Resource
-from flask_httpauth import HTTPBasicAuth
-
-import queries
-import backend as db
-from authentication import auth
-
-'''
+#!/usr/bin/env python3
+"""
 Main app
-'''
-app = Flask(__name__)
-
-api = Api(app)
-
-@api.resource("/runnumber/get")
-class getRunNumber(Resource):
-    @auth.login_required
-    def get(self):
-        rowRes = []
-        try:
-            db.perform_query(queries.getRunNum, {}, rowRes)
-        except Exception as e:
-            err_obj, = e.args
-            print("Exception:", err_obj.message)
-            resp = flask.make_response(flask.jsonify({"Exception": err_obj.message}))
-            return resp
-        print(rowRes)
-        resp = flask.make_response(flask.jsonify(rowRes))
-        return resp
 
-@api.resource("/runnumber/getnew")
-class getNewtRunNumber(Resource):
-    @auth.login_required
-    def get(self):
-        rowRes = []
-        try:
-            db.perform_transaction(queries.incrementRunNum, {})
-            db.perform_query(queries.getRunNum, {}, rowRes)
-        except Exception as e:
-            err_obj, = e.args
-            print("Exception:", err_obj.message)
-            resp = flask.make_response(flask.jsonify({"Exception": err_obj.message}))
-            return resp
-        print(rowRes)
-        resp = flask.make_response(flask.jsonify(rowRes))
-        return resp
+To control set the following environment variables:
+DATABASE_URI - URI for sqlalchemy to use
+DEBUG - set to TRUE for flask debugging
```

Comment:
Can you add the run start counter ENV to the note here?

Reply:
done

Reply:
I'm not seeing this in the latest commit....